File: controle/controle_funcionario.php
<?php
require_once '../dao/dao_funcionario.php';
require_once '../model/funcionario.php';

$op = $_POST["op"];

$funcionario = new Funcionario();
$funcionario->setNome($_POST["nome"]);
$funcionario->setCargo($_POST["cargo"]);
$funcionario->setEmail($_POST["email"]);
$funcionario->setLogin($_POST["login"]);
$funcionario->setSenha($_POST["senha"]);
$funcionario->setAtivo(true);

$daoFuncionario = new DaoFuncionario();

try {
    if ($op == "salvar") {
        $daoFuncionario->salvar($funcionario);
        echo "Sucesso";
    }
    else if ($op == "editar") {
        $daoFuncionario->editar($funcionario);
        echo "Sucesso";
    }
    else if ($op == "login") {
        $id = $daoFuncionario->buscar($funcionario);
        if($id != null)
        {
            session_start(); 
            $_SESSION["admin"] = $funcionario->getLogin();//Define que existe um usuário logado
            $_SESSION["id"] = $id; //Define que existe um usuário logado
            echo "Sucesso";
        }
    }
    else if ($op == "buscabusca") {
        $funcionarios = $daoFuncionario->busca_por_busca($funcionario);
        echo json_encode($funcionarios);
    }
    else if ($op == "remover") {
        $daoFuncionario->remover($funcionario);
        echo "Sucesso";
    }
} catch (Throwable $th) {
    echo "Falha" . $th;
}

File: controle/ControleFuncionario.py
from flask import render_template, session, redirect, url_for, request, json, Blueprint
from dao.DaoFuncionario import DaoFuncionario
from model.Funcionario import Funcionario

funcionario = Blueprint("funcionario", "Biblioteca", template_folder="../view", static_folder="../estilo")
daoFuncionario = DaoFuncionario()


@funcionario.route("/Funcionario/")
def admin():
    if 'admin' in session:
        template = render_template("cabecalho.html", inicio='active')
        template += render_template("logado.html")
        template += render_template("admin.html")
        template += render_template("rodape.html")
        return template
    else:
        return redirect(url_for('entrada.index'))


@funcionario.route("/Funcionario/Cadastro/")
def cadastro():
    if 'admin' in session:
        template = render_template("cabecalho.html")
        template += render_template("logado.html")
        template += render_template("cadastro_funcionario.html")
        template += render_template("rodape.html")
        return template
    else:
        return redirect(url_for('entrada.index'))


@funcionario.route("/Funcionario/Cadastrar/", methods=['POST'])
def cadastrar():
    try:
        nome = request.form['nome'];
        login = request.form['login'];
        senha = request.form['senha'];
        email = request.form['email'];
        cargo = request.form['cargo'];

        funcionarioM = Funcionario(cargo=cargo, nome=nome, login=login, senha=senha, email=email)
        daoFuncionario.create(funcionarioM)
        return json.dumps({'status': 'OK'});
    except Exception as e:
        return json.dumps({'status': 'Erro'});


@funcionario.route("/Funcionario/Perfil/")
def perfil():
    if 'admin' in session:
        template = render_template("cabecalho.html")
        template += render_template("logado.html", perfil='active')
        template += render_template("perfil.html")
        template += render_template("rodape.html")
        return template
    else:
        return redirect(url_for('entrada.index'))

@funcionario.route("/Funcionario/Buscar/", methods=['POST'])
def busca_busca():
    try:
        busca = request.form['busca'];

        funcionarios =daoFuncionario.search_search(busca)

        j = []
        for l in funcionarios:
            j.append({
                'id': l.id,
                'nome': l.nome,
                'login': l.login,
                'email': l.email,
                'cargo': l.cargo,
            })
        return json.dumps(j)

    except Exception as e:
        return json.dumps({'status': 'Erro'});

@funcionario.route("/Funcionario/Remover/", methods=['POST'])
def remover():
    try:
        id = request.form['id'];

        funcionarioM =daoFuncionario.search_id(id)
        daoFuncionario.remove(funcionarioM)
        return json.dumps({'status': 'Ok'});

    except Exception as e:
        return json.dumps({'status': 'Erro'});